### ISSUE TITLE
kubermatic installer failed

### ISSUE THREAD

**Issue by nitinkansal1984** [kubermatic/kubermatic#6595]
**User Story**
I have launched a eks cluster with 2 nodes and tried the steps for installing kubermatic, it failed in the step as below

root@linuxvm:~# ./kubermatic-installer deploy   --config kubermatic.yaml   --helm-values values.yaml   --storageclass aws
INFO[06:05:40] 🛫 Initializing installer…                     edition="Community Edition" version=v2.16.2
INFO[06:05:40] 🚦 Validating the provided configuration…
WARN[06:05:40]    Helm values: kubermaticOperator.imagePullSecret is empty, setting to spec.imagePullSecret from KubermaticConfiguration
INFO[06:05:40] ✅ Provided configuration is valid.
INFO[06:05:41] 🧩 Deploying kubermatic stack…
INFO[06:05:41]    💾 Deploying kubermatic-fast StorageClass…
INFO[06:05:41]    ✅ StorageClass exists, nothing to do.
INFO[06:05:41]    📦 Deploying nginx-ingress-controller…
ERRO[06:05:41] ❌ Operation failed: failed to deploy nginx-ingress-controller: failed to check to Helm release: failed to check for an existing release: failed to list releases: Error: Kubernetes cluster unreachable: Get "https://XXXXXXXX.gr7.us-east-1.eks.amazonaws.com/version?timeout=32s": getting credentials: exec: executable aws not found

It looks like you are trying to use a client-go credential plugin that is not installed.

To learn more about this feature, consult the documentation available at:
      https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins.

**Acceptance criteria**

**Comment by nitinkansal1984** [comment]
I tried the other way by installing it through helm explained in youtube video. But that too failed.

I noticed couple of errors also in kubermatic api pods.
root@linuxvm:~# kubectl get pods -n kubermatic
NAME                                                    READY   STATUS             RESTARTS   AGE
kubermatic-api-85488798d7-7llpr                         0/1     CrashLoopBackOff   8          20m
kubermatic-api-85488798d7-flk5n                         0/1     CrashLoopBackOff   6          20m
kubermatic-dashboard-8546dc5589-5szcj                   1/1     Running            0          31m
kubermatic-dashboard-8546dc5589-hvq6x                   1/1     Running            0          31m
kubermatic-master-controller-manager-59c7b8d656-tpf57   1/1     Running            0          31m
kubermatic-operator-84d7958dfc-k8tlp                    1/1     Running            0          32m
root@linuxvm:~#

root@linuxvm:~# kubectl logs kubermatic-api-85488798d7-7llpr -n kubermatic
{"level":"info","time":"2021-02-24T07:05:24.940Z","caller":"cli/hello.go:36","msg":"Starting Kubermatic API (Community Edition)...","version":"v2.16.2"}
{"level":"fatal","time":"2021-02-24T07:05:26.539Z","caller":"kubermatic-api/main.go:125","msg":"failed to create an openid authenticator","issuer":"https://kubermatic.test.com/dex","oidcClientID":"kubermatic","error":"oidc: failed to decode provider discovery object: expected Content-Type = application/json, got \"text/html\": invalid character '<' looking for beginning of value"}
root@linuxvm:~#
